internal/bucket/bucket_test.go

```diff
@@ -69,14 +69,16 @@ func TestOpen(t *testing.T) {
 	}
 
 	for name, tc := range tests {
-		tc := tc
 		t.Run(name, func(t *testing.T) {
 			t.Parallel()
 
 			b, err := bucket.Open(context.Background(), tc.config)
+			// We should definitely close buckets when we are testing!
+			// This throws a lint error which should be ignored!
+			defer b.Close()
```

Comment:
Thank you. I noticed that this panic when b is nil, which is possible in another test I wrote where Open is supposed to return an error instead. I've ended up doing this:

```go
if b != nil {
	b.Close()
}
```

gocloud seems to be doing the same, see [here](https://github.com/google/go-cloud/blob/5ec203b7dc558ebb940d0847f8dae12699f1d873/blob/s3blob/s3blob_test.go#L477-L480).